lib/transaction/transaction.js

```diff
@@ -297,32 +293,32 @@ Transaction.prototype.inspect = function() {
 };
 
```

Comment:
If you have time please add doc for all the method.

Reply:
Let's do a different ticket for that. It is a great amount of work and maybe it is not the best moment to do it